File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.cpp
class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2) {
        string res1="";
        for(int i=0;i<word1.size();i++){
            res1+=word1[i];
        }
        cout<<res1<<endl;
         string res2="";
        for(int i=0;i<word2.size();i++){
            res2+=word2[i];
        }
        
        return res2==res1;
        
    }
};

File: 1662-check-if-two-string-arrays-are-equivalent/1662-check-if-two-string-arrays-are-equivalent.js
/**
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
var arrayStringsAreEqual = function(word1, word2) {
      let res1="";
        for(let i=0;i<word1.length;i++)
            res1+=word1[i];
        
         let res2="";
        for(let i=0;i<word2.length;i++)
            res2+=word2[i];
        
        
        return res2==res1;
    
};